eval/eval.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+eval_store="$TMPDIR/eval"
+
+# ---------------------------------------------------------------------
+
+echo "::group::Nix Evaluation"
+
+ci_json="$(nix eval .#__std.ci."$(nix eval --raw --impure --expr 'builtins.currentSystem')" --json)"
```

Comment:
```suggestion
ci_json="$(nix eval .#__std.ci'."$(nix eval --raw --impure --expr 'builtins.currentSystem')" --json)"
```

(after alternate std PR)